packages/app-js/src/index.tsx

```diff
@@ -37,32 +39,53 @@ type Injected = {
 type Props = ApiProps & AppProps & I18nProps;
 type State = {
   code: string,
+  customExamples: Array<Snippet>
   isRunning: boolean,
   logs: Array<Log>,
-  snippet: string
+  options: Array<Snippet>,
+  snippet: string,
+  snippetName: string
+};
+
+const customExample: Snippet = {
+  code: ``,
+  label: { color: 'orange', children: 'Custom', size: 'tiny' },
+  text: '',
+  value: ''
 };
 
 class App extends React.PureComponent<Props, State> {
   injected: Injected | null = null;
   state: State = {
     code: '',
+    customExamples: [],
     isRunning: false,
     logs: [],
-    snippet: ''
+    options: [],
+    snippet: '',
+    snippetName: ''
   };
 
   componentDidMount () {
-    const local = snippets.find(obj => obj.value === localStorage.getItem('app-js-snippet'));
+    const localData = {
+      examples: localStorage.getItem('polkadot-app-js-examples'),
```

Comment:
Rather use the `store` module. Interface is the same, but generally it is easier to use since it serializes/deserializes objects. (Which also means you can actually combine the different keys into a single object, as it makes sense)

Reply:
Is there enough benefit of using the store module over the way it's implemented?
I'm reading from the store once onMount and I'm updating localStorage only in three places, which works perfectly fine with getItem, setItem and the JSON methods.
Adding a module for that seems a bit of an overload